README.md

```diff
@@ -3,7 +3,7 @@
 [![npm version](https://badge.fury.io/js/thirteen.svg)](http://badge.fury.io/js/thirteen)
 
 # thirteen
-#### (13, XIII, 十三, trece, tredici, 열세, तेरह, ثلاثة عشر, тринадцать, treze, dreizehn, baker's dozen, 11012, שלוש עשרה)
+#### (13, XIII, 十三, trece, tredici, 열세, तेरह, ثلاثة عشر, тринадцать, treze, dreizehn, baker's dozen, 1101, שלוש עשרה, Hamahiru, tredici, mười ba)
```

Comment:
Important stuff

Reply:
for future scientific advancement, etc...

Reply:
DEFINITELY.